engine/collection/ingest/engine.go

```diff
@@ -227,11 +228,13 @@ func (e *Engine) onTransaction(originID flow.Identifier, tx *flow.TransactionBod
 		log.Debug().Msg("propagating transaction to cluster")
 
 		err := e.conduit.Multicast(tx, e.config.PropagationRedundancy+1, txCluster.NodeIDs()...)
-		if err != nil {
+		if err != nil && !errors.Is(err, network.EmptyTargetList) {
```

Comment:
changed the logic to use the customer error instead of running the filter and figuring out if the target IDs ends up being empty in the engine code.